.meta/dietpi-survey_report

```diff
@@ -673,6 +673,7 @@ shopt -s extglob
 	unset -v 'aSOFTWARE_NAME8_19[141]' # Spotify Connect Web
 	unset -v 'aSOFTWARE_NAME8_19[177]' # Firefox Sync Server
 	aSOFTWARE_NAME8_19[210]='MediaWiki'
+	aSOFTWARE_NAME8_19[211]='Homebridge'
```

Comment:
This needs to go below to the v8.20 block as `aSOFTWARE_NAME8_20[211]='Homebridge'`.